### ISSUE TITLE
.net Core Support

### ISSUE THREAD

**Issue by atrauzzi** [adamcaudill/libsodium-net#148]
Will this library work on .net Core?  If not, are there any plans to get it working?

**Comment by maxenko** [comment]
Also looking for an answer to this. Going to need it soon.

**Comment by SProst** [comment]
Would it be possible to instead target netstandard? https://blogs.msdn.microsoft.com/dotnet/2016/09/26/introducing-net-standard/

**Comment by damccull** [comment]
I agree with targetting netstandard instead of core. Ideally, go back as far as you can in version so it'll be forward compatible, but still work with older code.

**Comment by atrauzzi** [comment]
Yeah, netstandard is the right way to go.
